commands/plugin_bot.go

```diff
@@ -172,7 +172,7 @@ func YAGCommandMiddleware(inner dcmd.RunFunc) dcmd.RunFunc {
 			guildID = data.GuildData.GS.ID
 		}
 
-		if data.TriggerType == dcmd.TriggerTypeSlashCommands {
+		if data.TriggerType == dcmd.TriggerTypeSlashCommands && data.SlashCommandTriggerData.Interaction.Type != discordgo.InteractionApplicationCommandAutocomplete {
```

Comment:
why not  >
`if data.TriggerType == dcmd.TriggerTypeSlashCommands && data.SlashCommandTriggerData != nil && data.SlashCommandTriggerData.Interaction.Type != discordgo.InteractionApplicationCommandAutocomplete`